src/app/utils/validators/is-hex.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { AbstractControl } from '@angular/forms';
+
+export function isHexValidator(control: AbstractControl) {
+  if (!control.value) {
+    return null;
+  }
+  const hexRegex = new RegExp('^(0x)([0-9a-fA-F])');
```

Comment:
it will be better if we create a RegEx when the file is initialized and not every time the method is called

Reply:
updated. Please check if this is what you expected.